samples/sampler/stories/qa/ContextualPopupDecorator.js

```diff
@@ -26,7 +26,30 @@ const renderWidePopup = () => (
 );
 
 const renderTallPopup = () => (
-	<div style={{height: ri.scaleToRem(402)}}>This is a tall popup</div>
+	<div style={{height: ri.scaleToRem(402)}}>
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+		This is a tall popup.
+	</div>
```

Comment:
Seems ok to me. Another suggestion would be to position the text at the bottom of the popup, but it needs confirmation from someone else